config/default_records/plate_purposes/005_limber_purposes.yml

```diff
@@ -30,6 +30,9 @@ LTN Stock:
 LCMB Cherrypick:
   <<: *limber_input_96
   cherrypickable_target: true
+RVI Cherrypick:
+  <<: *limber_input_96
```

Comment:
You've set it as an input plate here, but input false in Limber.
Needs to be consistent